test/component/test_component.py

```diff
@@ -0,0 +1,78 @@
+import time
+import uuid
+from unittest import TestCase
+
+import pika
+from coverage.python import os
+from google.cloud import pubsub_v1
+
+RABBIT_AMQP = "amqp://guest:guest@localhost:35672"
+SUBSCRIPTION_PROJECT_ID = "project"
```

Comment:
Are `SUBSCRIPTION_PROJECT_ID` and `SUBSCRIPTION_NAME` being used here?

Reply:
I expect most of these can be trimmed down.